data/language/en-GB.txt

```diff
@@ -98,6 +98,7 @@ STR_0093    :Hybrid Coaster
 STR_0094    :Single Rail Roller Coaster
 STR_0095    :Alpine Coaster
 STR_0096    :Classic Wooden Roller Coaster
+STR_0097    :Maurer Steel Roller Coaster
```

Comment:
We need a generic, non-branded name for this, right? I think the dev team discussed it a long time ago, but I don't remember if something was picked for this. If not, "Mini Looping Roller Coaster" is my nomination.

Reply:
If it's meant to be the Maurer track, the obvious name is "Spinning Roller Coaster"

Reply:
That makes sense!

Reply:
Due to the potential uses as a Eurofighter or Vekoma, would "Compact Looping Roller Coaster" be a better name?  This would allow the track to be more flexible and would not lead users into believing it is purely a spinning coaster.

Reply:
the renaming later was no problem i thinked of this at least too.
the Inversions are only enable as you use cheats for maurer only there is most elements blocked without cheats.

**SET_FIELD(EnabledTrackPieces**, { TRACK_FLAT, TRACK_STRAIGHT, TRACK_STATION_END, TRACK_LIFT_HILL, TRACK_LIFT_HILL_STEEP, TRACK_FLAT_ROLL_BANKING, TRACK_SLOPE, TRACK_SLOPE_LONG, TRACK_SLOPE_STEEP_UP, TRACK_SLOPE_STEEP_DOWN, TRACK_SLOPE_CURVE, TRACK_SLOPE_CURVE_STEEP, TRACK_S_BEND, TRACK_CURVE_SMALL, TRACK_CURVE, TRACK_CURVE_LARGE, TRACK_HELIX_DOWN_BANKED_HALF, TRACK_HELIX_UP_BANKED_HALF, TRACK_BRAKES, TRACK_ON_RIDE_PHOTO, TRACK_SLOPE_VERTICAL, TRACK_SLOPE_CURVE_BANKED, TRACK_BLOCK_BRAKES, TRACK_SLOPE_ROLL_BANKING, TRACK_CURVE_VERTICAL, TRACK_SLOPE_CURVE_LARGE, TRACK_POWERED_LIFT}),
    
**SET_FIELD(ExtraTrackPieces,**  {TRACK_BOOSTER, TRACK_SLOPE_STEEP_LONG, TRACK_CORKSCREW, TRACK_BARREL_ROLL, TRACK_BARREL_ROLL, TRACK_QUARTER_LOOP, TRACK_HALF_LOOP, TRACK_HALF_LOOP_MEDIUM, TRACK_HALF_LOOP_LARGE, TRACK_VERTICAL_LOOP, TRACK_CORKSCREW_LARGE, TRACK_ZERO_G_ROLL, TRACK_ZERO_G_ROLL_LARGE}),
    SET_FIELD(CoveredTrackPieces, {}),

Reply:
> 

maybe if you want i had your pinfari model and can do it later too

Reply:
Makes sense.  I was unsure if renaming at a later date was a possibility.  I appreciate the information.

Reply:
> Due to the potential uses as a Eurofighter or Vekoma, would "Compact Looping Roller Coaster" be a better name? This would allow the track to be more flexible and would not lead users into believing it is purely a spinning coaster.

Well you have to decide what ride type it's supposed to be. If you want to use the same track type for multiple ride types, there should really be multiple RTDs with different names that just share track paint code.

The name should match how the ride is set up - if it's supposed to be a spinning coaster, the inversions should not be available without cheats, the stats, max height, build cost etc should all reflect the spinning coaster. If it's supposed to be a Eurofighter that stuff should be different.

I really don't think this track should be labelled a Eurofighter if it doesn't have the beyond vertical drop (I'm also not sure that this track best fits a Eurofighter, only the very earliest examples had this style). In terms of being a ride that isn't already in the game, and really should be, I think the Maurer/Gerstlauer spinner is a good choice.

Reply:
> > Due to the potential uses as a Eurofighter or Vekoma, would "Compact Looping Roller Coaster" be a better name? This would allow the track to be more flexible and would not lead users into believing it is purely a spinning coaster.
> 
> Well you have to decide what ride type it's supposed to be. If you want to use the same track type for multiple ride types, there should really be multiple RTDs with different names that just share track paint code.
> 
> The name should match how the ride is set up - if it's supposed to be a spinning coaster, the inversions should not be available without cheats, the stats, max height, build cost etc should all reflect the spinning coaster. If it's supposed to be a Eurofighter that stuff should be different.
> 
> I really don't think this track should be labelled a Eurofighter if it doesn't have the beyond vertical drop (I'm also not sure that this track best fits a Eurofighter, only the very earliest examples had this style). In terms of being a ride that isn't already in the game, and really should be, I think the Maurer/Gerstlauer spinner is a good choice.



as said the inversions are all not available in normal mode. "Maurer/Gerstlauer Roller Coaster" is not bad. It was planned as Maurer Sc3000 I just meant you could use it as one of the first models for gerstlauer eurofighter but you don't have to.

I will call it "Maurer/Gerstlauer Roller Coaster".

Reply:
> I will call it "Maurer/Gerstlauer Spinner".

I recommend just calling it "Spinning Roller Coaster" to be consistent with the game's naming convention. None of the vanilla rides mention the manufacturer name - and it isn't really needed, because the only other spinning coaster is called "spinning wild mouse" which is different enough.

Also, not having the manufacturer listed means that it could have both Maurer and Gerstlauer trains as options, which is consistent with the way rides like the hyper twister work - these don't really need to be separate ride types, just different trains as they are so similar.

Reply:
Agreed do not mention manufacturer names it should be generic.